xcode/Subconscious/Shared/Components/Common/Transclude/TranscludeButtonStyle.swift

```diff
@@ -14,14 +14,18 @@ struct TranscludeButtonStyle: ButtonStyle {
     @Environment(\.isEnabled) private var isEnabled
     @Environment(\.colorScheme) var colorScheme
     
-    var color: Color
-    
     func makeBody(configuration: Configuration) -> some View {
         configuration.label
             .padding(.vertical, AppTheme.unit3)
             .padding(.horizontal, AppTheme.unit3)
             .expandAlignedLeading()
-            .background(color)
+            .background(
+                .background.opacity(
+                    colorScheme == .dark
+                    ? 0.1
+                    : 0.5
+                )
+            )
```

Comment:
I have no justification for this other than eyeballing it, see the screenshots in the PR description.